themes/phenomic-theme-base/src/components/DefaultHeadMeta/index.js

```diff
@@ -23,10 +12,12 @@ export default class Layout extends Component {
       pkg,
     } = this.context.metadata
 
+    /* eslint-disable react/jsx-key */
     return (
-      <div className={ styles.layout }>
+      <div hidden>
```

Comment:
Haha. This will upset IE. We should add a notice about html5 shim if user want to support IE < 9